### PR TITLE
ams account create arm template

### PR DESCRIPTION
**Pull request by IngridAtMicrosoft**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by IngridAtMicrosoft** [comment]
#sign-off

**Comment by mgblythe** [comment]
@bmoore-msft - this one's ready to go

File: 101-media-services-account-create/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Create a Media Services Account using a template
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2F101-media-services-create%2Fazuredeploy.json" target="_blank">
```
Comment (bmoore-msft) [inline 101-media-services-account-create/README.md:3]: See the contribution guide or other doc samples for how to construct a proper README.md for the repo

File: 101-media-services-account-create/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "mediaServiceName": {
```
Comment (bmoore-msft) [inline 101-media-services-account-create/azuredeploy.json:5]: add a defaultValue using uniqueString()

File: 101-media-services-account-create/metadata.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (bmoore-msft) [inline 101-media-services-account-create/metadata.json:1]: add the $schema property

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
See the pipeline results for the best practice violations that need to be addressed

**Comment by bmoore-msft** [comment]
Closing abandoned request - please reopen if you can address the feedback.

**Comment by mgblythe** [comment]
@mumian as FYI

**Comment by mumian** [comment]
@IngridAtMicrosoft @mgblythe  -  there is an existing quickstart template - 101-media-services-create.  I have another PR refreshing the template - https://github.com/Azure/azure-quickstart-templates/pull/8627.

**Comment by IngridAtMicrosoft** [comment]
> @IngridAtMicrosoft @mgblythe - there is an existing quickstart template - 101-media-services-create. I have another PR refreshing the template - #8627.
@mumian That is a different ARM template created by a customer.  I am supplying two others.  Although this one is similar, it is the one from Microsoft and Media Servicess.

**Comment by mumian** [comment]
@IngridAtMicrosoft Azure Quickstart templates is a community contributed repository. The two templates are very similar. Both create a storage account resource and a media service resource.  We should update the existing template instead of creating a new one.

**Pull request closed without merging**